tools/build.js

```diff
@@ -24,14 +24,14 @@ const GLOBALS = {
   'firebase/firestore': 'firebase',
   'firebase/functions': 'firebase',
   'firebase/storage': 'firebase',
-  'angularfire2': 'angularfire2',
-  'angularfire2/auth': 'angularfire2.auth',
-  'angularfire2/database': 'angularfire2.database',
-  'angularfire2/database-deprecated': 'angularfire2.database_deprecated',
-  'angularfire2/firestore': 'angularfire2.firestore',
-  'angularfire2/functions': 'angularfire2.functions',
-  'angularfire2/storage': 'angularfire2.storage',
-  'angularfire2/messaging': 'angularfire2.messaging',
+  '@angular/fire': 'angularfire2',
```

Comment:
Should these be changed?

Reply:
We've decided to keep the UMD names as they are; not worth breaking anyone using them.